nextcord/member.py

```diff
@@ -887,6 +887,23 @@ async def move_to(self, channel: Optional[VocalGuildChannel], *, reason: Optiona
         """
         await self.edit(voice_channel=channel, reason=reason)
 
+    async def disconnect(self, *, reason: Optional[str] = None) -> None:
+        """|coro|
+
+        Disconnects a member from the voice channel they are connected to.
+
+        You must have the :attr:`~Permissions.move_members` permission to
+        use this.
+
+        This raises the same exceptions as :meth:`edit`.
```

Comment:
I feel it would be better to have these here regardless

Reply:
> I feel it would be better to have these here regardless

have what here regardless?

Reply:
the errors listed

Reply:
i just copy pasted the line from the move_to thing, so if its not specific enough here, it shouldn't be at move_to either.

Reply:
Oki doki, feel free to resolve